packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -72,6 +74,10 @@ $ms-Persona-presenceSizeXl: 28px;
   display: flex;
   align-items: center;
 
+  &:focus {
+    outline: 1px solid $focusedBorderColor;
```

Comment:
We only show focus borders when `.ms-Fabric.is-focusVisible` is set. You can search the code for:

```css
@include focus-border();
```

Include that mixin and it will add the &:focus selector for you.